test/re/frida/MethodTest.java

```diff
@@ -39,6 +40,99 @@ public void replacementPropagatesExceptions() {
         thrown.expectMessage("Already dead");
         badger.die();
     }
+    
+    static private String failString = null;
+    static private void Fail( String msg )
+    {
+      failString = msg;
+    }
+    
+    @Test
+    public void TestNewInterface() {
+        loadScript("var X509TrustManager = Java.use('javax.net.ssl.X509TrustManager');" +
+                "try{" +
+                "  var tm = X509TrustManager.$new();" +
+                "}catch(e){" + 
```

Comment:
For the JS we should stick to [semistandard](https://github.com/Flet/semistandard), the style we're using in the implementation code. This is almost there, just missing spaces on each side of the curly braces and after `catch`.